bruges/rockphysics/rpmodels.py

```diff
@@ -0,0 +1,561 @@
+# -*- coding: utf-8 -*-
+"""
+===================
+rpmodels.py
+===================
+
+Rock physics models.
+References are mentioned in docstrings of individual functions.
+Docstrings follow numpy/scipy convention, can be changed
+to be consistent with the rest of bruges, if needed.
+Input units are not SI; velocities in m/s,
+elastic moduli GPa, density g/cm3, etc.
+
+Alessandro Amato del Monte, October 2019
+"""
+import numpy as np
+
+
+def vels(Kdry,Gdry,K0,D0,Kf,Df,phi):
+    '''
+    Calculate velocities and densities of saturated rock via Gassmann equation.
+
+    Parameters
+    ----------   
+    Kdry, Gdry : float or array_like
+        Dry rock bulk & shear modulus in GPa.
+    K0, G0 : float or array_like
+        Mineral bulk & shear modulus in GPa.
+    Kf, Df : float or array_like
+        Fluid bulk modulus in GPa and density in g/cc.
+    phi : float or array_like
+        Porosity.
+
+    Returns
+    -------
+    vp,vs,rho : float or array_like
+        P-wave, S-wave velocities and density of saturated rock,
+        in m/s and g/cc.
+    '''
+    # converts all inputs to SI (density in kg/m3 and moduli in Pa)
+    Kdry *=1e9
+    Gdry *=1e9
+    K0 *=1e9
+    D0 *=1e3
+    Kf *=1e9
+    Df *=1e3
+    rho = D0*(1-phi)+Df*phi
+    with np.errstate(divide='ignore', invalid='ignore'):
+        K = Kdry + (1-Kdry/K0)**2 / ( (phi/Kf) + ((1-phi)/K0) - (Kdry/K0**2) )
+        vp = np.sqrt((K+4/3*Gdry)/rho)
+        vs = np.sqrt(Gdry/rho)
+    return vp, vs, rho/1e3, K/1e9
+
+
+def critpor(K0, G0, phi, phi_c=0.4):
```

Comment:
All these functions behave in a similar way, i.e. they return K_dry and G_dry (bulk and shear elastic moduli). These two things always go together I suppose, because you then use them togetjher with density to reconstruct velocities. In the case of critical porosity it is true, you could split the function in two bu in other cases they are interlinked; so I would keep it like this for consistency with other functions and also to avoid the proliferation of too many functions.